src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -209,17 +210,24 @@ private void handleDelete(String commandResult) throws CommandException {
         try {
             index = Integer.parseInt(commandResult.substring(DELETE_COMMAND_INDEX));
             if (focusListPanel.getCandidate().equals(logic.getFilteredCandidateList().get(index - 1))) {
-                if (!focusListPanelPlaceholder.getChildren().isEmpty()) {
-                    focusListPanelPlaceholder.getChildren().remove(0);
-                }
-                FocusCard focusCard = new FocusCard(null, null);
-                focusListPanelPlaceholder.getChildren().add(focusCard.getRoot());
+                clearFocusCard();
             }
         } catch (Exception e) {
             return;
         }
     }
 
+    /**
+     * Clear Focus Card
+     */
+    public void clearFocusCard() {
+        if (!focusListPanelPlaceholder.getChildren().isEmpty()) {
+            focusListPanelPlaceholder.getChildren().remove(0);
+        }
+        candidateListPanel = new CandidateListPanel(logic.getFilteredCandidateList());
```

Comment:
@tiewweijian what does this do ah!

Reply:
Ehh cause now when we clear the `FocusCard`, there is still a greyish background on it ! 
This line will instantiate a new CandidateListPanel ! So the background looks nicer!